server/src/service.rs

```diff
@@ -458,4 +474,42 @@ impl Service {
 
         Ok(signal_emitter)
     }
+
+    pub async fn remove_client(&self, old_owner: &str) -> Option<OwnedObjectPath> {
+        self.clients.lock().await.remove(old_owner)
```

Comment:
Make this return the session directly instead of the session path

Reply:
Removed remove_client()